src/draw/grouping.typ

```diff
@@ -13,6 +13,41 @@
 
 #import "transformations.typ": move-to
 
+/// Hide an element.
+///
+/// Hidden elements act like normal elements, despite not getting
+/// rendered and not producing a bounding box.
```

Comment:
"Hide*s*..."

"Hidden elements are not drawn to the canvas, are ignored when calculating bounding boxes and discarded by `merge-path`. All other behaviours remain the same as a non-hidden element."